modules/ROOT/pages/active-directory-or-ldap-authentication.adoc

```diff
@@ -259,14 +259,86 @@ The Active Directory configuration has been tested in single domain configuratio
 
 == Troubleshooting
 
-If necessary, you can enable JAAS debug mode by editing your configuration file and adding the following line: `debug=true`
+If necessary, you can edit your configuration to have additional logs (please note, a server restart will be required for the new configuration to be taken into account, though):
 
-On Active Directory, a common error code is:
-`LDAP: error code 49 - 80090308: LdapErr: DSID-0C0903A9, comment: AcceptSecurityContext error, data 52e, v1db1`.
-This error code can have several root causes:
+* Enable JAAS debug mode by editing your JAAS configuration file (e.g. `<TOMCAT_HOME>/server/conf/jaas.cfg`) and adding the following line: `debug=true`
+* Enable java's https://docs.oracle.com/javase/8/docs/technotes/guides/security/jsse/ReadDebug.html[SSL/TLS connection debug mode] by editing your application server's environment setting file (e.g. `<TOMCAT_HOME>/setup/tomcat-templates/setenv.sh(.bat)`) and adding the following system property to the java options: `-Djavax.net.debug=all`
+
+[discrete]
+==== Common error examples
```

Comment:
```suggestion
=== Common error examples
```

I suggest you to never use [discrete], it removes the creation of the anchor. And you should not jump a title level, it can lead to a confusing rendering.